src/state/gearOptimizerSlice.js

```diff
@@ -136,331 +28,84 @@ export const gearOptimizerSlice = createSlice({
           selectedSpecialization: firstUppercase(action.payload),
           status: WAITING,
         };
-        state.skills = [];
       }
     },
     changeExpertMode: (state, action) => {
       if (action.payload) {
         // just change expertMode to true
         return {
           ...state,
-          control: {
-            ...state.control,
-            expertMode: action.payload,
-          },
+          expertMode: action.payload,
         };
       }
       // wipe state and force usr to select a template
       return {
-        ...INITIALSTATE,
-        control: {
-          expertMode: action.payload,
-          list: [],
-          progress: 0,
-          selectedCharacter: null,
-          status: WAITING,
-        },
-        ar: state.ar,
-        omnipotion: state.omnipotion,
-      };
-    },
-    changeAR: (state, action) => {
-      state.ar = action.payload;
-    },
-    changeTraitLine: (state, action) => {
-      state.traits.lines[action.payload.index] = action.payload.value;
-    },
-    changeTraits: (state, action) => {
-      state.traits.selected[action.payload.index] = action.payload.selected;
-    },
-    changeGeneric: (state, action) => {
-      state[action.payload.toChange] = action.payload.value;
-    },
-    changeDistributionVersion: (state, action) => {
-      state.distribution.version = action.payload;
-    },
-    changeDistributionNew: (state, action) => {
-      state.distribution = {
-        ...state.distribution,
-        values2: { ...state.distribution.values2, [action.payload.index]: action.payload.value },
-      };
-    },
-    changeTextBoxes: (state, action) => {
-      state.distribution = {
-        ...state.distribution,
-        textBoxes: {
-          ...state.distribution.textBoxes,
-          [action.payload.index]: action.payload.value,
-        },
+        ...state,
```

Comment:
works a bit weirdly right now (settings persist from expert mode into easy mode but cannot be changed) but I want to refactor that anyways

Reply:
yeah, I didnt touch the switching logic for expertMode since I expected that you had something in mind there